docs/reference/composition.md

```diff
@@ -787,7 +787,9 @@ so:
 
 1. Use a `ToCompositeFieldPath` patch to patch from your source composed
    resource to the XR. Typically you'll want to patch to a status field or an
-   annotation.
+   annotation. Note: The field you are patching to needs to be defined in the 
+   XR definition. For instance, if you are patching to `.status.foo` then the definition
```

Comment:
This is true for status fields but not for metadata.labels and metadata.annotations which are very often used as "scratchpads" to copy information from one composite to another.   Maybe clarify that this note only applies to status fields and not to annotations or labels.

Reply:
Good point. I'll update.